webapp/src/main/webapp/css/vitro.css

```diff
@@ -255,7 +255,7 @@ ul.language-dropdown {
 }
 ul.language-dropdown li#language-menu {
     background: url(../images/arrowDownOverAccount.gif) right 9px no-repeat;
-    min-width:150px;
+    min-width:20em;
```

Comment:
This does not result in a fix for me. The width of the dropdown seems to be the same size required by the text: 
- "Select a language"

I still render the same as the previous screenshot. It would be nice to have a fix here... but if it is not worth the time, we can move forward.

Reply:
Here is a trade-off that works for the dropdown, but moves the unexpanded menu slightly to the left:
```
 ul.language-dropdown {
     position: relative;
+    min-width: 160px;
 }
 ul.language-dropdown li#language-menu {
     background: url(../images/arrowDownOverAccount.gif) right 9px no-repeat;
-    min-width:20em;
     position: relative;
     margin-right: -10px !important;
 }
```

Reply:
<img width="332" alt="Screen Shot 2020-05-28 at 10 30 33 PM" src="https://user-images.githubusercontent.com/10748475/83221366-e1fb4900-a132-11ea-8ce2-e97a87791b3e.png">

This results in the menu appearing to jump to the left when opening it, is that what you see?

Reply:
Does this work for you?

```
ul.language-dropdown {
    position: relative;
    min-width: 160px;
}
ul.language-dropdown li#language-menu {
    background: url(../images/arrowDownOverAccount.gif) right 9px no-repeat;
    position: relative;
    margin-right: -10px !important;
    min-width: 170px;
}
```

<img width="329" alt="Screen Shot 2020-05-28 at 10 33 53 PM" src="https://user-images.githubusercontent.com/10748475/83221603-79f93280-a133-11ea-9261-9148ee59c484.png">

Reply:
@gneissone : your suggested css update renders nicely. Let's go with it.

Reply:
Gentlemen, it seems like the fix works in Chrome but in Firefox.

![image](https://user-images.githubusercontent.com/46490666/84152417-98fc9c00-aa32-11ea-8bdc-a519a25190aa.png)

Reply:
@nicalico: do you have any interest in tweaking the css to make this work?
Otherwise, I would suggest we move this change forward, and refine later.

Reply:
I could try to tweak it, but it would have to wait a few days.